__tests__/components/editor/property/PropertyResourceTemplate.test.js

```diff
@@ -18,7 +22,7 @@ describe('<PropertyResourceTemplate />', () => {
         },
       ],
     },
-    reduxPath: ['resourceTemplate:test'],
+    reduxPath: ['http://id.loc.gov/ontologies/bibframe/title'],
```

Comment:
for those of us lagging behind on the redux store (me!) - what is this URL representing?  The propertyURI?  It might make it more obvious to use a fake one like 'http://example.org/propertyUriGoesHere'.

Reply:
I think it's important to use a real one so we can show that title can have more than one possible sub-resources. In this case VariantTitle, TransliteratedTitle or WorkTitle.